apps/api/src/test-data/validateTestData.spec.ts

```diff
@@ -234,6 +235,18 @@ describe('buildTestData', () => {
         });
     });
 
+    describe('the test data for users', () => {
```

Comment:
If validate invariants is on all aggregates, can't we do this check polymorphically so that we don't have to update this test again?